### PR TITLE
Minetest update to 0.4.15, stop using -ffast-math compiler flag

### PR DESCRIPTION
**Pull request by nigels-com**
Fix for minetest on OSX issue:
https://github.com/minetest/minetest/issues/4274

### CODE REVIEW

**Comment by neoascetic** [comment]
You have a wrong verstion in the title :)

**Comment by nigels-com** [comment]
Oops, sorry.

File: minetest.rb [hunk 1]
```diff
@@ -3,21 +3,15 @@ class Minetest < Formula
...
 
     resource "minetest_game" do
-      url "https://github.com/minetest/minetest_game/archive/0.4.14.tar.gz"
-      sha256 "eebc2830d0e2f431f648691a0072fbb773d000c56deaf09ddc5e7416096cdf85"
+      url "https://github.com/minetest/minetest_game/archive/0.4.15.tar.gz"
+      sha256 "f583ea55f511bbcc106c60f7d488ec3fe6b961efc3fd648603b6acd0189d7d8a"
     end
   end
 
-  bottle do
```
Comment (mistydemeo) [inline minetest.rb]: For future reference: you don't have to worry about doing this, since new bottles are being added when it's merged.

**Comment by mistydemeo** [comment]
Thanks!

**Comment by nigels-com** [comment]
Works great!  Thanks.

**Pull request closed without merging**